001-quickstart-passwordless/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// ------------------------------------------------------------
+
+// <using_directives> 
+using Microsoft.Azure.Cosmos;
+using Azure.Identity;
+// </using_directives>
+
+// <client_credentials> 
+// New instance of CosmosClient class
+using CosmosClient client = new(
+    "https://<your-cosmosdb-account-name>.documents.azure.com:443/",
```

Comment:
Should this use an environmental variable for the account endpoint? I'm not sure what practice is used in other similar articles:

```suggestion
    accountEndpoint: Environment.GetEnvironmentVariable("COSMOS_ENDPOINT")!
```